src/core/jit/variables.rs

```diff
@@ -0,0 +1,25 @@
+use std::collections::HashMap;
+
+use serde::Deserialize;
+
+/// Variables store
+#[derive(Debug, Deserialize)]
+pub struct Variables<Input>(HashMap<String, Input>);
```

Comment:
Move it to `model.rs`. Later we can build a module `model` to split the models further into smaller files.

Reply:
done